mentor_classifier_api/src/mentor_classifier_api/__init__.py

```diff
@@ -49,4 +49,8 @@ def create_app():
 
     app.register_blueprint(healthcheck_blueprint, url_prefix="/classifier/healthcheck")
 
+    from mentor_classifier_api.blueprints.fetch import fetch_blueprint
```

Comment:
need to update the import, also might as well change the name of the actual blueprint var to `trainingdata_blueprint` to follow the existing convention